### DISCUSSION TITLE
Is it possible to switch out the animated "loading" icon?

### DISCUSSION THREAD

**Discussion by carwei** [lowdefy/lowdefy#1411] [Questions and Help] [answered]
Is it possible to have a custom animated icon to replace the default "loading" icon?
I looked here but don't think it's covered: https://docs.lowdefy.com/hosting-files
Thank you!

**Comment by Gervwyk** [comment, marked as answer]
In v4 there is no loading icon only a more generic progress bar. The icon cannot be changed in v3.
